app/views/lessons/show.html.erb

```diff
@@ -27,11 +27,13 @@
   <%= link_to 'Book Lesson', new_lesson_booking_path(@lesson) %>
 <% end %>
 
+<% if @reviews != nil %>
```

Comment:
refactored?

Reply:
REFACTORED:
<img width="1440" alt="Schermata 2020-10-28 alle 08 15 06" src="https://user-images.githubusercontent.com/70261924/97404019-b5bc4200-18f5-11eb-9688-35268b89cc0d.png">

<% if @reviews.nil? %>
<h5>Missing link to go on new review</h5>
<% else %>
<% @reviews.each do |review| %>
  <h3><%= review.title %></h3>
  <p><%= review.rating %></p>
  <p><%= review.description %></p>
<% end %>
<% end %>

Tested it on a new lesson with no reviews.